extensions/OpenMod.Extensions.Economy.Abstractions/OpenMod.Extensions.Economy.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Also update target framework in https://github.com/openmod/openmod/tree/main/templates and https://github.com/openmod/openmod/tree/main/tools

Reply:
Done